js/controller.js

```diff
@@ -859,7 +867,24 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
      * @param multiAudio.multiAudio {Array} - list of objects with data for each audio
      */
     onMultiAudioFetched: function(event, multiAudio) {
-    //  TODO: code for a video with multiaudio should be here
+      this.state.multiAudio = multiAudio;
+    },
+
+    /**
+     * @fires OO.EVENTS.SET_CURRENT_AUDIO
+     * @param {String} id - the id of the audio track to activate
+     */
+    setCurrentAudio: function(id) {
+      this.mb.publish(OO.EVENTS.SET_CURRENT_AUDIO, id);
```

Comment:
Should the event be fired after the current audio is set?

Reply:
This event are fired after, for example, user has clicked to audio button and chose new audio.